samcli/lib/test_runner/generate_env_vars.py

```diff
@@ -0,0 +1,94 @@
+import yaml
+from typing import List
+from samcli.commands.exceptions import FailedArnParseException
+
+
+class FargateRunnerArnMapGenerator:
```

Comment:
Is a class needed?

Reply:
Not necessarily, I was just following OOP practice for consistency and to make it easier to extend in the future.